code/hub.dm

```diff
@@ -6,7 +6,7 @@
 #else
 	hub_password = "SORRYNOPASSWORD"
 #endif
-	name = "/tg/ Station 13"
+	name = "Cocktail Party Station 13"

```

Comment:
A small group of folks that play and mod games. The reason for the fork was to create a SS13 catering to our play style. However was planning on sharing any fixes to bugs in the shared code base. Hence the grammar fix, I'm new to GitHub and SVNs, hence the balls up. Sorry.